entrypoint/versionbump/versionbump_test.go

```diff
@@ -34,11 +34,11 @@ var _ = Describe("Version Bump Template Upgrade:", func() {
 	It("Test Bump version", func() {
 		Template(TestTemplate{
 			TestCombination: &RunCmd{
-				Run: []TestMap{
+				Run: []customflag.TestMap{
 					{
-						Cmd:                  TestMapTemplate.Cmd,
-						ExpectedValue:        TestMapTemplate.ExpectedValue,
-						ExpectedValueUpgrade: TestMapTemplate.ExpectedValueUpgrade,
+						Cmd:                  customflag.Tm.Cmd,
```

Comment:
Can we not use shorthand here: `TestMap` instead of `Tm`

Reply:
it will duplicate the var, so we cant. 
but i can try a better name